patients/get_allergies.php

```diff
@@ -16,11 +16,11 @@
     ?>
     <table class="class1">
         <tr class="header">
-    <th><?php echo htmlspecialchars(xl('Title'), ENT_NOQUOTES); ?></th>
-    <th><?php echo htmlspecialchars(xl('Reported Date'), ENT_NOQUOTES); ?></th>
-    <th><?php echo htmlspecialchars(xl('Start Date'), ENT_NOQUOTES); ?></th>
-    <th><?php echo htmlspecialchars(xl('End Date'), ENT_NOQUOTES); ?></th>
-    <th><?php echo htmlspecialchars(xl('Referrer'), ENT_NOQUOTES); ?></th>
+    <th><?php echo htmlspecialchars(xlt('Title'), ENT_NOQUOTES); ?></th>
+    <th><?php echo htmlspecialchars(xlt('Reported Date'), ENT_NOQUOTES); ?></th>
+    <th><?php echo htmlspecialchars(xlt('Start Date'), ENT_NOQUOTES); ?></th>
+    <th><?php echo htmlspecialchars(xlt('End Date'), ENT_NOQUOTES); ?></th>
+    <th><?php echo htmlspecialchars(xlt('Referrer'), ENT_NOQUOTES); ?></th>
```

Comment:
xlt() is basically shorthand for:
```
htmlspecialchars(xl('label'))
```
So, in this, case, remove the htmlspecialchars stuff in above lines, for example:
```
<th><?php echo xlt('Referrer'); ?></th>
```